internal/pkg/encoding/json/schema/schema.go

```diff
@@ -153,8 +156,8 @@ func processErrors(errs []*jsonschema.ValidationError, parentIsSchemaErr bool) e
 			if strings.HasSuffix(e.InstanceLocation, "/required") && e.Message == "expected array, but got boolean" {
 				continue
 			}
-			// The schema may contain a non-standard testConnection definition: {type: button}, so the error should be ignored.
-			if path == "properties.test_connection.type" {
+			// JSON schema may contain empty enums, in dynamic selects.
+			if strings.HasSuffix(e.InstanceLocation, "/enum") && e.Message == "minimum 1 items required, but found 0 items" {
```

Comment:
V druhom commite som vylepsil kod, aby testy presli:
- :red_circle: odstranil som z tohto miesta ignorovanie `test_connection` chyby, kedze je to fatalna chyba, tak som to sice ignoroval, ale ziadna dalsia validacia sa uz nerobila - tj. cela validacia bola v tomto pripade vypnuta, musel som to vyriesit inac, vid nizsie
- :green_circle: pridal som tu ignorovanie prazdnej `enum` definicie, ktora sa pouziva pri dynamickych select boxes.